src/components/CommandBar/CommandBarReview.tsx

```diff
@@ -88,6 +91,15 @@ function CommandBarReview({ stepBack }: { stepBack: () => void }) {
           <CommandBarDivider />
         </>
       )}
+      {reviewValidationError && (
+        <>
+          <p className="px-4 py-2 text-red-500 text-sm">
+            <span className="font-bold">Error: </span>
```

Comment:
Does this need to say "Error"? Maybe not a big deal but the UX when the user tries to Extrude is to see this. It's not an error for them, because they haven't had an opportunity to add a Length or a To value yet.

Reply:
Good point. Would you want it not red too?